src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int upbothside;
+    private int downbothside;
+    private int height;
```

Comment:
Class variable names should be descriptive and follow the camelCase naming convention. Instead of 'upbothside' and 'downbothside', consider 'upperBase' and 'lowerBase' to make it clear that these are the lengths of the trapezoid's bases.